src/main/kotlin/no/nav/tilleggsstonader/sak/journalføring/JournalføringService.kt

```diff
@@ -196,8 +196,15 @@ class JournalføringService(
     private fun lagreSøknadOgNyeBarn(
         journalpost: Journalpost,
         behandling: Behandling,
+        stønadstype: Stønadstype,
     ) {
-        lagreSøknad(journalpost, behandling.id)
+        lagreSøknad(journalpost, behandling.id, stønadstype)
+        if (skalLagreBarn(stønadstype)) {
```

Comment:
Denne kan nå erstattes med 
```suggestion
        if (stønadstype.gjelderBarn()) {
```

Husk å fjerne `skalLagreBarn`